<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>450</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QListWidget" name="leftListWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>180</width>
      <height>400</height>
     </rect>
    </property>
   </widget>
   <widget class="QListWidget" name="rightListWidget">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>0</y>
      <width>180</width>
      <height>400</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="leftToRightButton">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>0</y>
      <width>240</width>
      <height>40</height>
     </rect>
    </property>
    <property name="text">
     <string>&gt;</string>
    </property>
   </widget>
   <widget class="QPushButton" name="rightToLeftButton">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>360</y>
      <width>240</width>
      <height>40</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;</string>
    </property>
   </widget>
   <widget class="QPushButton" name="addLeftButton">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>120</y>
      <width>120</width>
      <height>40</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt; Добавить </string>
    </property>
   </widget>
   <widget class="QPushButton" name="deleteButton">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>80</y>
      <width>240</width>
      <height>40</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string>Удалить</string>
    </property>
   </widget>
   <widget class="QPushButton" name="clearAllButton">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>240</y>
      <width>240</width>
      <height>40</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt; Очистить &gt;</string>
    </property>
   </widget>
   <widget class="QPushButton" name="editButton">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>40</y>
      <width>240</width>
      <height>40</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string>Изменить</string>
    </property>
   </widget>
   <widget class="QPushButton" name="sortLeftButton">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>280</y>
      <width>120</width>
      <height>40</height>
     </rect>
    </property>
    <property name="text">
     <string> &lt; Сортировать</string>
    </property>
   </widget>
   <widget class="QPushButton" name="addRightButton">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>120</y>
      <width>120</width>
      <height>40</height>
     </rect>
    </property>
    <property name="text">
     <string>Добавить &gt;</string>
    </property>
    <property name="checkable">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="addAllButton">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>160</y>
      <width>240</width>
      <height>40</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt; Добавить &gt;</string>
    </property>
    <property name="checkable">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="sortRightButton">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>280</y>
      <width>120</width>
      <height>40</height>
     </rect>
    </property>
    <property name="text">
     <string>Сортировать &gt;</string>
    </property>
   </widget>
   <widget class="QPushButton" name="sortAllButton">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>320</y>
      <width>240</width>
      <height>40</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt; Сортировать &gt;</string>
    </property>
   </widget>
   <widget class="QPushButton" name="clearRightButton">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>200</y>
      <width>120</width>
      <height>40</height>
     </rect>
    </property>
    <property name="text">
     <string>Очистить &gt;</string>
    </property>
   </widget>
   <widget class="QPushButton" name="clearLeftButton">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>200</y>
      <width>120</width>
      <height>40</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt; Очистить</string>
    </property>
   </widget>
   <widget class="QWidget" name="horizontalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>400</y>
      <width>601</width>
      <height>51</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <property name="leftMargin">
      <number>0</number>
     </property>
     <property name="rightMargin">
      <number>0</number>
     </property>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>30</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QCheckBox" name="viewModeAllButton">
       <property name="font">
        <font>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Режим пиктограмм</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QCheckBox" name="MultiSelectionButton">
       <property name="font">
        <font>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Множественное выделение</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_3">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
